pdg/src/graph.rs

```diff
@@ -155,7 +158,7 @@ impl Node {
         let fn_ = function;
         write!(
             f,
-            "{kind}{sep}{src}{sep}=>{sep}{dest}{sep}@{sep}{bb_stmt}:{sep}fn {fn_};"
+            "{kind}{sep}{src}{sep}=>{sep}{dest}{sep}@{sep}{bb_stmt}:{sep}fn {fn_};{sep}{debug}"
```

Comment:
```suggestion
            "{kind}{sep}{src}{sep}=>{sep}{dest}{sep}@{sep}{bb_stmt}:{sep}fn {fn_};{sep}{debug};"
```

End with a `;` to make syntax highlighting better.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/566/commits/7e3105321218231f871a8110609f922335763f79.